wrappers/swift/LogManager.swift

```diff
@@ -0,0 +1,160 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper over ODWLogManager which manages the telemetry logging system.
+public class LogManager {
+
+    /**
+    Initializes the telemetry logging system with the specified tenant token and custom configuration.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - config: A `Dictionary` that contains a custom configuration.
+
+    - Returns:
+        - A `Logger` instance.
+    */
+    public static func initForTenant(tenantToken: String!, config: [AnyHashable: Any]? = nil) -> Logger? {
+        if let odwLogger = ODWLogManager.initForTenant(tenantToken, withConfig: config) {
+            return Logger(logger: odwLogger)
+        } else {
+            return nil
+        }
+    }
+
+    /**
+    Retrieves a new instance of `Logger` for logging telemetry events. Initializes the telemetry logging system,
+    if not already initialized, with the default configuration using the specified tenant token and source.
+
+    - Parameters:
+        - tenantToken: A `String` that contains the tenant token.
+        - withSource: A `String` that contains the name of the source of events.
+
+    - Returns:
+        - A `Logger` instance that points to the logger for the specified tenant token and source.
+    */
+    public static func loggerWithTenant(tenantToken: String!, withSource source: String! = "") -> Logger? {
```

Comment:
Do the parameters on these functions need to be optionals?

Reply:
No. Its a mistake. I have corrected it.
Thanks for pointing it out.